deploy.py

```diff
@@ -309,7 +304,16 @@ def checked_command_output(self, cmd, env=None, cwd=None):
   def update_rustc_if_needed(self):
     target_toolchain_fullstring = "stable"
     with open("rust-toolchain", "r", encoding="utf-8") as f:
-      target_toolchain_fullstring = f.readline().strip()
+      content = f.readlines()
+      if len(content) == 1:
+        # Old format, only the build is stored
+        target_toolchain_fullstring = f.readline().strip()
```

Comment:
I see I made a mistake here.
This should be `content[0].strip()` instead of `f.readline().strip()`. The whole file has already been read in line 307.

Reply:
True, that's better. Done